internal/service/organization/data_source_organization.go

```diff
@@ -63,11 +64,11 @@ func dataSourceMongoDBAtlasOrganizationRead(ctx context.Context, d *schema.Resou
 		return diag.FromErr(fmt.Errorf("error setting `is_deleted`: %s", err))
 	}
 
-	if err := d.Set("links", flattenOrganizationLinks(organization.Links)); err != nil {
+	if err := d.Set("links", flattenOrganizationLinks(organization.GetLinks())); err != nil {
 		return diag.FromErr(fmt.Errorf("error setting `is_deleted`: %s", err))
 	}
 
-	d.SetId(organization.ID)
+	d.SetId(*organization.Id)
```

Comment:
does it also work with organization.GetId() ?

Reply:
yes, updated